cmd/fleetctl/get.go

```diff
@@ -28,12 +29,22 @@ type specGeneric struct {
 	Spec    interface{} `json:"spec"`
 }
 
-func defaultTable() *tablewriter.Table {
-	table := tablewriter.NewWriter(os.Stdout)
+func defaultTable(writer ...io.Writer) *tablewriter.Table {
+	w := writerOrStdout(writer)
+	table := tablewriter.NewWriter(w)
 	table.SetRowLine(true)
 	return table
 }
 
+func writerOrStdout(writer []io.Writer) io.Writer {
```

Comment:
Not that I can think. This was mostly because I wanted to add the ability to add a writer to `defaultTable` without having to change it everywhere, so I added the `...` which then translated to this. But should be just one.

Reply:
I just went ahead and made everything use `printTable` and also fixed this.